source/fundamentals/data-formats/documents.txt

```diff
@@ -14,7 +14,10 @@ Documents
 Overview
 --------
 
-A MongoDB **document** is a data structure that contains key/value fields in
+In this guide, you can learn how you can work with **documents** in the
```

Comment:
```suggestion
In this guide, you can learn how to use **documents** in the
```

Suggestion: similar to previous suggestion, "you can learn how you can" I think should be replaced.